core/GlobalState.cc

```diff
@@ -1915,6 +1915,7 @@ unique_ptr<GlobalState> GlobalState::deepCopy(bool keepId) const {
     result->censorForSnapshotTests = this->censorForSnapshotTests;
     result->sleepInSlowPathSeconds = this->sleepInSlowPathSeconds;
     result->requiresAncestorEnabled = this->requiresAncestorEnabled;
+    result->lspExperimentalFastPathEnabled = this->lspExperimentalFastPathEnabled;
```

Comment:
@froydnj one last commit in case you want to take a look at the command line option code

https://github.com/sorbet/sorbet/pull/5808/commits/e802719f7d0b277378773653b8e529c63bce0f4f

i'm testing on Stripe's codebase right now